### DISCUSSION TITLE
DAI 2023-2024 - Practical work 2 - Richard Aurélien and Pirakas Anthon

### DISCUSSION THREAD

**Discussion by Lyrafll** [heig-vd-dai-course/heig-vd-dai-course#149] [Show and tell]
The objective is to create a server side application and a client side one.
Both will be done with the same JAR, juste executed/called differently

Server:
Will let user connect to it, identify themselves and let them answer a quizz (already present in the server, JSON file)

Client : 
Will connect to server, enter his name and will be able to answer a quizz

The quizz will contain different type of question:
- True/False
- Multiple choices
- Free Answer

**Comment by apirakas** [comment]
Our repo last commit is here https://github.com/apirakas/DAI-PW2/commit/e3d471e8bc22fe7150320c052ad69e0291d46557
